flow_action_components/CollectionProcessors/force-app/main/default/classes/GenerateCollectionReport.cls

```diff
@@ -29,14 +29,21 @@ global with sharing class GenerateCollectionReport {
             if (inputCollection != null && !inputCollection.isEmpty() && shownFields != null) {
                 reportString += 'Collection Type: ' + inputCollection[0].getSObjectType().getDescribe().getName() + '\n\n';
                 List<String> shownFieldsArray = shownFields.replaceAll('[^a-zA-Z0-9\\,\\_\\.]', '').split(',');
+                for(String field : shownFieldsArray){
+                    system.debug('Shown Field: ' + field);
+                }
```

Comment:
Can we get rid of these? They were for just debugging.  It won't let the PR requestor remove code. @alexed1

Reply:
Hmm. What happens when you try removing them in your local project. Then add and commit? Removing is no different from adding in fit world...

Reply:
I got it figured out. :-)